internal/pkg/cli/validate.go

```diff
@@ -122,6 +122,17 @@ func validateDomainName(val interface{}) error {
 	return nil
 }
 
+func validatePath(val interface{}) error {
```

Comment:
Did we decide to not check that the file exists in the validator? It's ok if not, or there are other technical issues.

Reply:
Done.